tests/tests/test_update_control.py

```diff
@@ -193,3 +194,95 @@ def test_update_control_with_broken_map(
         # Wait for the device to reject it and fail.
         deploy.check_expected_status("finished", deployment_id)
         deploy.check_expected_statistics(deployment_id, "failure", 1)
+
+    def test_update_control_with_expiring_control_map(
+        self, enterprise_no_client, valid_image,
+    ):
+        """
+        Run an update, in which the download stage takes longer than the
+        expiry time of the control map.
+
+        In other words, test MEN-5096.
+
+        This is done by having an Artifact script pause in Download for a given <tbd> time, then continue on until
+        """
+
+        u = User("", "bugs.bunny@acme.org", "whatsupdoc")
+        cli = CliTenantadm(containers_namespace=enterprise_no_client.name)
+        tid = cli.create_org("enterprise-tenant", u.name, u.pwd, "enterprise")
+        tenant = cli.get_tenant(tid)
+        tenant = json.loads(tenant)
+        ttoken = tenant["tenant_token"]
+
+        auth = Authentication(name="enterprise-tenant", username=u.name, password=u.pwd)
+        auth.create_org = False
+        auth.reset_auth_token()
+        devauth = DeviceAuthV2(auth)
+
+        enterprise_no_client.new_tenant_client("control-map-test-container", ttoken)
+        device = MenderDevice(enterprise_no_client.get_mender_clients()[0])
+        devauth.accept_devices(1)
+
+        deploy = Deployments(auth, devauth)
+
+        devices = list(
+            set([device["id"] for device in devauth.get_devices_status("accepted")])
+        )
+        assert 1 == len(devices)
+
+
+        with tempfile.NamedTemporaryFile(prefix="Download_Leave_01_", mode="w") as sleep_script:
+            # Sleep for x min > UpdateControlMapExpirationTimeSeconds used in the tests
```

Comment:
```suggestion
            # Sleep for 1 min > UpdateControlMapExpirationTimeSeconds used in the tests
```
?

Reply:
I'm just removing it all together. Not a good comment.